providers/aws_ec2/impl.go

```diff
@@ -228,6 +242,17 @@ func (prov *Provider) start(mach *providers.Machine) bool {
 	}
 
 	log.Printf("EC2 instance '%s' is running\n", *inst.InstanceId)
+
+	// We're running, we can attach the volumes
+	for _, v := range prov.AttachVolumes {
+		v.InstanceId = inst.InstanceId
+		_, err := prov.Ec2.AttachVolume(ctx, v)
+		if err != nil {
+			fmt.Printf("Error in attaching volume: %v\n", err)
+			return false
```

Comment:
I'm wondering if we should do something else here. Current behaviour leaves the machine lingering, which feels wrong. LazySSH is supposed to manage machines for you, and this can potentially rack up costs.

The two alernatives I can think of:

 - Terminate the instance.
 - Log the error, but otherwise don't fail.

@MarcoPolo What do you think?

Reply:
Oh! that's surprising. I didn't look carefully at the code, my apologies.

I think the correct behavior should be that it terminates the instance if this fails. Since 1. it is literally a cheaper option, and 2. code that sshs into this machine expects that drive(s) to be there. So it would be better to fail at the beginning than enter a weird state later on.

I'll make another PR that has `start` return an error, and then we can terminate the instance depending on the type of error we get back (à la `errors.Is`).